frontend/src/model/beta/settings/EntityItem.tsx

```diff
@@ -0,0 +1,94 @@
+import ClearIcon from '@mui/icons-material/Clear'
+import GroupsIcon from '@mui/icons-material/Groups'
+import PersonIcon from '@mui/icons-material/Person'
+import { Autocomplete, Chip, IconButton, TableCell, TableRow, TextField, Tooltip, Typography } from '@mui/material'
+import _ from 'lodash-es'
+import { Dispatch, SetStateAction } from 'react'
+import Loading from 'src/common/Loading'
+
+import { useGetModelRoles } from '../../../../actions/model'
+import { CollaboratorEntry, ModelInterface } from '../../../../types/v2/types'
+
+type EntityItemProps = {
+  entity: CollaboratorEntry
+  accessList: CollaboratorEntry[]
+  setAccessList: Dispatch<SetStateAction<CollaboratorEntry[]>>
```

Comment:
Personal preference would be for this to be agnostic of what the "event" is being used for `onAccessListChange: (value: CollaboratorEntry[]) => void`